src/libraries/Descriptor.sol

```diff
@@ -503,6 +503,10 @@ library Descriptor {
         }
     }
 
+    function isSpecialCharacter(bytes1 b) private pure returns (bool) {
+        return b == '"' || b == "\u000c" || b == "\n" || b == "\r" || b == "\t";
+    }
```

Comment:
These new strings are now multiple bytes long so I don't think this byte by byte parsing will match